var/spack/repos/builtin/packages/gnuplot/package.py

```diff
@@ -43,24 +41,93 @@ class Gnuplot(Package):
     homepage = "http://www.gnuplot.info"
     url      = "http://downloads.sourceforge.net/project/gnuplot/gnuplot/5.0.1/gnuplot-5.0.1.tar.gz"
 
+    version('5.0.5', 'c5e96fca73afbee4f57cbc1bfce6b3b8')
     version('5.0.1', '79b4f9e203728f76b60b28bcd402d3c7')
 
+    variant('wx',      default=False,
+            description='Activates wxWidgets terminal')
+    variant('gd',      default=True,
+            description='Activates gd based terminal')
+    variant('cairo',   default=True,
+            description='Activates cairo based terminal')
+    variant('x11',     default=True,
```

Comment:
We usually call this variant `+X`, right? I don't care which we use, but we should probably be consistent across packages. Also, this should default to False like it does in every other package, at least until the concretizer is fixed.